packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyStateData.ts

```diff
@@ -11,7 +11,7 @@ import type {
 
 export const headers: ResourcesHeaders = {
   description:
-    'Store your own custom Linux images, enabling you to rapidly deploy Linode Compute Instances preconfigured with the software and settings you require.',
+    'Store custom Linux images so you can rapidly deploy compute instances that are preconfigured with what you need.',
```

Comment:
am no english major but this is reads pretty bloated
maybe more succinct?

```suggestion
    'Store custom Linux images to rapidly deploy compute instances preconfigured with what you need.',
```

Reply:
Our Technical Writer was good with that change!